eng/pipelines/libraries/helix.yml

```diff
@@ -18,6 +18,10 @@ parameters:
   SuperPmiCollectionName: ''
 
 steps:
+  - script: $(Build.SourcesDirectory)/.dotnet/dotnet build -c Release $(Build.SourcesDirectory)/src/tests/Common/XUnitLogChecker/XUnitLogChecker.csproj /p:OutDir=$(Build.SourcesDirectory)/artifacts/tests/Common/XUnitLogChecker
```

Comment:
@ivdiazsa @ViktorHofer is this the way we want to build the log-checker for libs?   Seems like it might be better to just plug it into to an existing MSBuild execution rather than complicate the YML.   I could imagine it fitting into libs.test, libs.pretest, sendtohelixhelp.proj ... which do you think makes the most sense?

Reply:
I mentioned to Eric the other day that it would be great if we can compile XUnitLogChecker in the CoreCLR build. That way, it's only built once, and we can reuse it in the Libraries tests. For now, this was the way I was able to get it to build, so it's definitely temporary.

Reply:
That works too - if there's a natural place to put it that's building similar managed code - I just want to be sensitive to putting more complexity into the pipelines.

Reply:
Yes, this doesn't belong into the pipeline but I see that this was already addressed. Building this tool in libs makes more sense than in the runtime tree. Building it as part of the libs subset doesn't mean that you can't build it only once.